pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -53,6 +65,35 @@ func SingleClientDHCPv6Server(
 	return nil
 }
 
+func (h *DHCPv6Handler) createConnection() (*FilteredConn, error) {
+	// no connection provided by the user, create a new one
+	addr := &net.UDPAddr{
+		IP:   net.IPv6unspecified,
+		Port: dhcpv6.DefaultServerPort,
+	}
+	udpConn, err := server6.NewIPv6UDPConn("", addr)
+	if err != nil {
+		return nil, err
+	}
+
+	packetConn := ipv6.NewPacketConn(udpConn)
```

Comment:
The `FilteredConn` structure seems to encapsulate this `packetConn`, but here you expose it.
As that structure has already a `Close` method, adding `New`/`Create` and putting all the details there may fit for a proper abstraction.

Reply:
Can you please elaborate? I don't understand your suggestion.

Reply:
@EdDev that's what you meant?:)

Reply:
done. Moved the method to conn.go.